### ISSUE TITLE
BUG: `xx not in ["a", "b"]` doesn't work

### ISSUE THREAD

**Issue by iredmail** [NikolaLohinski/gonja#16]
- `var in ["value1", "value2"]` works.
- `not (var in ["value1", "value2"])` works
- `var not in ["value1", "value2"]` doesn't work, the result is always `False`

```
package main

import (
	"os"

	"github.com/nikolalohinski/gonja/v2"
	"github.com/nikolalohinski/gonja/v2/exec"
)

func main() {
	template, err := gonja.FromString("in: {{ name in ['a'] }}\nnot in: {{ name not in ['a'] }}\nnot (in): {{ not name in ['a'] }}\n")
	if err != nil {
		panic(err)
	}

	data := exec.NewContext(map[string]interface{}{
		"name": "bob",
	})

	if err = template.Execute(os.Stdout, data); err != nil {
		panic(err)
	}
}
```
It prints:
```
in: False        // correct and expected
not in: False    // WRONG. BUG
not (in): True   // correct and expected
```

**Comment by NikolaLohinski** [comment]
Hi @iredmail and thanks for the report. it should be fixed with https://github.com/NikolaLohinski/gonja/pull/17 which I just released in `v2.2.1`. 

Feel free to reopen the issue if needed.

Cheers

**Comment by iredmail** [comment]
Dear @NikolaLohinski 

Thanks for the fix. Sample code in my report works fine with v2.2.1. :)
